presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -208,6 +208,17 @@ public static Identity extractAuthorizedIdentity(Optional<Identity> optionalAuth
         return identity;
     }
 
+    /**
+     * Creates a session identity.
+     * <p>
+     * Checks if the user is non-null, add header, add extraCredentials and add groups to build a session identity.
```

Comment:
You should also use more the link tag on the descriptions to point to important classes like the Session Identity